docs/api/flink/Constructor.md

```diff
@@ -20,9 +27,41 @@ Format: `ST_GeomFromGeoJSON (GeoJson:string)`
 Since: `v1.2.0`
 
 SQL example:
+
 ```sql
-SELECT ST_GeomFromGeoJSON(polygontable._c0) AS polygonshape
-FROM polygontable
+SELECT ST_GeomFromGeoJSON('{
```

Comment:
Please also show an example that uses only the geometry field of a GeoJSON file.